### ISSUE TITLE
In Dark mode, `<Toast />` close button's focus indicator color lacks 3:1 contrast ratio.

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#583]
![9de03e40-7f5e-11ec-a196-f77e1b8afb8a](https://user-images.githubusercontent.com/3180185/159584051-671c0743-9a2b-4eb4-a1ea-bb5cf0129039.png)

---
<dl><dt>Description</dt><dd> The state of an ACTIVE user interface component does not have at least a 3 to 1 luminosity contrast ratio with either the inner or outer adjacent background.
In Dark Theme - The focus indicator color of the 'Dismiss' control is failing cc ratio 3 to 1. Below is the details of 'Confirmation' toast
Foreground: #FFFFFF
Background: #D1FFA4
CC ratio: 1.1:1</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The visual state of an active user interface component MUST have sufficient contrast of 3 to 1 with the adjacent background. Exceptions exist.

HOW TO FIX: 
Fix this issue by adjusting the visual focus indicator or state indicator (e.g. selected, checked, pressed, etc.) of the user interface component and/or background to increase the contrast with either the inner or outer adjacent background to at least 3 to 1.

REFERENCE: 
WCAG Understanding document: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

BACKGROUND: 
People who have low vision or are colorblind may have difficulty perceiving that an element is interactive or what its state is - e.g. whether it has keyboard focus, is selected/checked/pressed, etc. - if the contrast between the element boundaries and/or state indicators and its background or adjacent colors is insufficient. When an interactive element's visual boundaries and state indicators have adequate contrast, people who have low vision or are colorblind are more likely to be able to perceive which elements on the screen are interactive and what their current state is.</dd><dt>Source code</dt><dd><pre><code><button type="button" class="Toast__dismiss" aria-label="Dismiss"><div aria-hidden="true" class="Icon Icon--close"><svg xmlns="http://www.w3.org/2000/svg" overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z" vector-effect="non-scaling-stroke" fill="currentColor"></path></svg></div></button></code></pre></dd><dt>Checkpoint</dt><dd>Non-Text Contrast - States of User Interface Components (1.4.11.b)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Toast</dd></dl>
